test/manipulation.test.js

```diff
@@ -1870,7 +1870,7 @@ describe('manipulation', function() {
     });
 
     bdd.itIf(connectorCapabilities.supportStrictDelete !== false &&
-      connectorCapabilities.cloudantCompatible,
+      connectorCapabilities.cloudantCompatible !== false,
```

Comment:
it should be `&&` I think, otherwise the connector(not cloudant) which doesn't support strict delete will still run this test due to `connectorCapabilities.cloudantCompatible !== false` is `true`